packages/react-devtools-shared/src/backend/index.js

```diff
@@ -64,10 +65,6 @@ export function initBackend(
   ];
 
   const attachRenderer = (id: number, renderer: ReactRenderer) => {
-    // skip if already attached
-    if (renderer.attached) {
-      return;
-    }
```

Comment:
Frontend side of devtools needs `renderer-attached` event to be emitted. With this check this will never happen and `agent` will not receive operations from hook.

Reply:
cc @mondaychen

Are there any cases I am missing where we really need this check?

Reply:
Ah yes, I think this was something I missed. Let's remove it now.